### PR TITLE
Modified FileSystemResultsWriter.java

### PR DESCRIPTION
**Pull request by AutomatedOwl**
Changed OutputStream object to BufferedOutputStream in order to solve https://github.com/allure-framework/allure-java/issues/197 .
Tested and verified.

### CODE REVIEW

File: allure2-model-api/src/main/java/io/qameta/allure/FileSystemResultsWriter.java [hunk 1]
```diff
@@ -37,7 +36,7 @@ public void write(TestResult testResult) {
                 : generateTestResultName(testResult.getUuid());
         createDirectories(outputDirectory);
         Path file = outputDirectory.resolve(testResultName);
-        try (OutputStream os = Files.newOutputStream(file, CREATE_NEW)) {
+        try (OutputStream os = new BufferedOutputStream(new FileOutputStream(file.toString()))) {
```
Comment (baev) [inline allure2-model-api/src/main/java/io/qameta/allure/FileSystemResultsWriter.java]: why not using `new BufferedOutputStream(Files.newOutputStream(file, CREATE_NEW))` ?

**Review by baev — CHANGES_REQUESTED** [review]

File: allure2-model-api/src/main/java/io/qameta/allure/FileSystemResultsWriter.java (hunk 1 above)
Comment (AutomatedOwl) [reply allure2-model-api/src/main/java/io/qameta/allure/FileSystemResultsWriter.java]: I debugged both operations, the former causes the allure results exeception. It's reproduced here: https://github.com/AutomatedOwl/allure-testng-result-write-bug

Comment (baev) [reply allure2-model-api/src/main/java/io/qameta/allure/FileSystemResultsWriter.java]: I see, let me check it 

PS you need to add `aspectjweaver` dependency to maven surefire plugin in order to run it (I got no aspectjweaver:1.9.1 in my local maven repo, so build fails)

Comment (AutomatedOwl) [reply allure2-model-api/src/main/java/io/qameta/allure/FileSystemResultsWriter.java]: Updated pom.xml at https://github.com/AutomatedOwl/allure-testng-result-write-bug

Comment (baev) [reply allure2-model-api/src/main/java/io/qameta/allure/FileSystemResultsWriter.java]: Yeah, I guess we need to use blocking io, so your solution is fine. Please replace `file.toString()` with `file.toFile()` and update other methods `write` as well

Comment (AutomatedOwl) [reply allure2-model-api/src/main/java/io/qameta/allure/FileSystemResultsWriter.java]: @baev changes done

**Review by baev — APPROVED** [review]

**Comment by AutomatedOwl** [comment]
@baev thanks, what's missing now for it to be merged?

**Comment by baev** [comment]
@AutomatedOwl waiting for Jenkins (will fix it myself)

BTW it would be so nice if you remove wildcard imports from your patch

**Comment by baev** [comment]
@AutomatedOwl thanks!

**Comment by AutomatedOwl** [comment]
@baev amazing, thanks! first commit but not the last :)

**Pull request merged**